### ISSUE TITLE
W002 being triggered even though X-UA-Compatible meta tag exists

### ISSUE THREAD

**Issue by Jrizzi1** [twbs/bootlint#229]
I ran bootlint on my project and it gave the following console err

> bootlint:  W002  `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility > modes

i checked and my project has the meta tag but includes chrome as well as IE in the content attribute
```html
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
```
when I change the meta tag to 
```html
<meta http-equiv="X-UA-Compatible" content="IE=edge">
```
the error no longer shows, seems like this is a bug, doesnt it?

**Comment by juthilo** [comment]
Google has ceased support for Chrome Frame, meaning it is no longer being updated and should no longer be used. Don't know how that affects any decisions with regard to Bootlint though.

**Comment by cvrebert** [comment]
Yeah:
> [we’ve decided to retire Chrome Frame, and will cease support and updates for the product in January 2014.](http://blog.chromium.org/2013/06/retiring-chrome-frame.html)

Given that, and also https://github.com/validator/validator/pull/12 , I'm inclined to deem this WontFix.
@hnrch02 What's your opinion?

**Comment by hnrch02** [comment]
Duplicate of #152. Yeah, WontFix.
